bitnami/mysql/values-production.yaml

```diff
@@ -85,10 +85,11 @@ master:
     ## If true, use a Persistent Volume Claim, If false, use emptyDir
     ##
     enabled: true
+    mountPath: /bitnami/mysql
```

Comment:
The reasoning for setting the mountPath here is for flexibility with other images, right?

Reply:
Yes... I mean, the user can also use its own configuration file using a different path for data. Therefore, the mountPath for the data volume should be flexible.